spec/mailers/user_mailer_spec.rb

```diff
@@ -432,4 +432,74 @@
       end
     end
   end
+
+  describe "invited to collection notification" do
+    let(:collection) { create(:collection) }
+    let(:otheruser) { create(:user) }
+    let(:work) { create(:work, authors: [otheruser]) }
+
+    let(:email) { UserMailer.invited_to_collection_notification(otheruser.id, work.id, collection.id).deliver }
```

Comment:
I think it was fine to have this as `subject`, despite what Hound said about the placement. (We only just started using Hound for RSpec, so we might want to tweak that particular setting. cc: @redsummernight )

Reply:
Was this the [LeadingSubject](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecleadingsubject) warning? It's ok to have the subject first even if it references the other lets, as they're lazily evaluated. Docs example:
```rb
  subject { described_class.new(params) }
  let(:params) { blah }
```

Reply:
Is this referring to a different line? I'm slightly confused.

Reply:
Oh, sorry, I thought this had been `subject(:email) { UserMailer.invited_to_collection_notification(otheruser.id, work.id, collection.id).deliver }`, and Hound had commented on it with the LeadingSubject warning, and you'd replied to Hound to say you didn't think moving it would work. I must have confused it with a different line.

However, this should get changed to `subject` like the others specs.

Reply:
Got it, will do.

Reply:
It looks like `let(:email)` still needs to be changed to `subject(:email)`.

Reply:
Ok, done.